app/models/user.rb

```diff
@@ -96,6 +96,7 @@ class User < Principal
   has_many :emoji_reactions, dependent: :destroy
   has_many :reminders, foreign_key: "creator_id", dependent: :destroy, inverse_of: :creator
   has_many :remote_identities, dependent: :destroy
+  has_many :user_auth_provider_links, dependent: :destroy
```

Comment:
:green_circle: Also here, because it's possible:

```suggestion
  has_many :user_auth_provider_links, dependent: :destroy
  has_many :authentication_providers, through: :user_auth_provider_links
```